### ISSUE TITLE
[Device Support Request] Philips Hue Go (1st Gen)

### ISSUE THREAD

**Issue by TheJulianJES** [zigpy/zha-device-handlers#447]
**Is your feature request related to a problem? Please describe.**
I'm currently unable to see the PhilipsOnOffCluster for the Philips Hue Go LED (2015 on Amazon.de: https://www.amazon.de/dp/B00UEMFGQM/) in the ZHA gui.
This light supports RGB + cold and warm white -> so similar to normal hue color bulbs(?)

**Describe the solution you'd like**
To be able to change the default power on state of the Philips Hue light.
Perhaps, it would be enough to just add the model to https://github.com/zigpy/zha-device-handlers/blob/dev/zhaquirks/philips/lct.py (?)
(Wasn't able to test this yet, as I'm currently unable to get this setting to work on other hue bulbs -> Additional context)

**Device signature**
(If the log of a repairing process is also required, please let me know)
```
{
  "node_descriptor": "<NodeDescriptor byte1=1 byte2=64 mac_capability_flags=142 manufacturer_code=4107 maximum_buffer_size=71 maximum_incoming_transfer_size=45 server_mask=0 maximum_outgoing_transfer_size=45 descriptor_capability_field=0>",
  "endpoints": {
    "11": {
      "profile_id": 49246,
      "device_type": "0x0210",
      "in_clusters": [
        "0x0000",
        "0x0003",
        "0x0004",
        "0x0005",
        "0x0006",
        "0x0008",
        "0x0300",
        "0x1000",
        "0xfc01"
      ],
      "out_clusters": [
        "0x0019"
      ]
    },
    "242": {
      "profile_id": 41440,
      "device_type": "0x0061",
      "in_clusters": [
        "0x0021"
      ],
      "out_clusters": [
        "0x0021"
      ]
    }
  },
  "manufacturer": "Philips",
  "model": "LLC020",
  "class": "zigpy.device.Device"
}
```

**Additional context**
In https://github.com/zigpy/zha-device-handlers/issues/399#issuecomment-675149667 there's currently kind of an "issue" that is kind of related to this.
Basically, I'm also unable to set the power_on_state attribute through the PhilipsOnOffCluster for lights like the LCT010 (with the latest firmware).
If this is really a bug in zhaquirks, I can also file an issue.
